apps/prairielearn/src/sync/course-db.ts

```diff
@@ -266,6 +266,8 @@ export interface AssessmentAllowAccess {
   active: boolean;
   timeLimitMin: number;
   password: string;
+  showClosedAssessment: boolean;
```

Comment:
Was this noticed when you removed some of the lodash mentions?

Reply:
Yep - for `_.get(foo, 'bar')`, TypeScript won't check that `foo` has a `bar property, but naturally it will when you just do `foo.bar`.